symphony/app/fbcnms-projects/hub/app/components/Main.js

```diff
@@ -49,6 +51,13 @@ function NavBarItems() {
       icon={<RouterIcon />}
       onClick={() => {}}
     />,
+    <NavListItem
+      key={2}
+      label="Workflows"
+      path="/hub/workflows/defs"
```

Comment:
Could we make the tab lead to `/hub/workflows` then add another switch/router for `/defs`  and the other pages? The `/hub/workflows` route can redirect you to `/hub/workflows/defs` by default but I'd like that logic to maybe live separately, since this is kind of a sub-app.
DM me if I'm not clear, I can provide a code sample or mode details.